exe/manageiq-cross_repo

```diff
@@ -31,6 +31,10 @@ opts = Optimist.options do
     Can also be passed as a REPOS environment variable.
   EOS
 
+  opt :script_cmd, <<~EOS, :type => :string, :default => ENV["SCRIPT_CMD"].presence || ""
+    Optional, a command string for running the specs.  Defaults to `bundle exec rake`.
```

Comment:
An alternative to this that I just thought of was `--test-rake-task` (or something... naming is hard...)

So instead of needing to do `SCRIPT_CMD="bundle exec rake spec:my_spec", it could just be `RAKE_TASK=spec:my_spec` and it is appended on to the existing string if it exists.

Reply:
My hunch is the generality of SCRIPT_CMD will come in handy sooner or later... 
Example: you're modifying lib/manageiq/environment.rb and want to demonstrate that `bin/update `still works in plugins.

LGTM either way.